save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationTestsMenu.kt

```diff
@@ -110,7 +101,17 @@ private fun organizationTestsMenu() = FC<OrganizationTestsMenuProps> { props ->
             fetchTestSuitesSourcesSnapshotKeys()
         }
     }
-
+    showTestSuiteSourceCreationModal(
+        isTestSuiteSourceCreationModalOpen,
+        props.organizationName,
+        {
+            setIsTestSuitesSourceCreationModalOpen(false)
+            setIsSourceCreated { !it }
```

Comment:
do we need `isTestSuiteSourceCreated` as additional dependency if we call `fetchTestSuitesSources` which updates `content`?

Reply:
As far as there were `dependencies = arrayOf(props.organizationName)`, no request was performed. I decided to add extra dependency - `isTestSuiteSourceCreated`

Reply:
But when we update `isTestSuiteSourceCreated`, we trigger `fetchTestSuitesSources` which updates `contest` which updates the table. Or I miss something?

Reply:
Yep, this is what happens, afaiu.

Reply:
so, it means that `isTestSuiteSourceCreated` is not needed as dependency in table because it will be re-rendered after `contest` is updated

Reply:
You are right, will remove it.

Reply:
Done